### PR TITLE
Images in section row are now full width

### PR DESCRIPTION
**Pull request by zacharyfmarion**
Think this looks better, but let me know if you think otherwise.

### CODE REVIEW

File: src/components/PostListRow.js [hunk 1]
```diff
@@ -47,8 +58,8 @@ const styles = StyleSheet.create({
     marginBottom: 3,
   },
   thumbnail: {
-    marginRight: -10,
-    marginLeft: -10,
+    //marginRight: -10,
```
Comment (blakejkaplan) [inline src/components/PostListRow.js]: Mind deleting instead of commenting out?

File: src/components/PostListRow.js [hunk 2]
```diff
@@ -22,6 +24,15 @@ const styles = StyleSheet.create({
     borderBottomWidth: 7,
     borderColor: '#DDDDDD',
   },
+  postRowImageContainer: {
+    flex: 1,
+    flexDirection: 'column',
+    justifyContent: 'center',
+    //paddingTop: 20,
```
Comment (blakejkaplan) [inline src/components/PostListRow.js]: Same with this one

**Comment by zacharyfmarion** [comment]
Removed the comments and fixed the issue with abnormally-sized images. I can squash the commits into one if you think that would be cleaner.

**Comment by blakejkaplan** [comment]
Looks good.

**Pull request merged**